src/components/ReviewsByCategory.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import { getReviewsByCategory } from "../api";
+import ReviewCard from "./ReviewCard";
+
+export default function ReviewsByCategory() {
+  const [reviewListByCategory, setReviewListByCategory] = useState([]);
+  let [searchParams] = useSearchParams();
+
+  useEffect(() => {
+    getReviewsByCategory(searchParams.get("category")).then((data) => {
+      setReviewListByCategory(data);
+    });
+  }, [searchParams]);
+
+  return (
+    <ul className="review-container">
+      {reviewListByCategory.map((review) => {
+        return <ReviewCard review={review} />;
+      })}
+    </ul>
+  );
+}
```

Comment:
We can simply your code a little bit in this case. Reviews by category should be the same component reused as the UI looks the same as all reviews. We can do this by reusing the previously implemented get all reviews function to filter by category (using axios params)